exercises/practice/robot-simulator/.meta/example.gd

```diff
@@ -0,0 +1,40 @@
+class RobotSimulator:
+
+var _pos: Vector2i
```

Comment:
I think we don't need to use a class here, we can create an instance of a script using `new()` (or `instance()`). But, if we use a class, then the rest of the file has to be indented (everything after `class RobotSimulator:` line).

Reply:
I think an exercise where we did use a nested class like this would be helpful for students so they explicitly have a chance to set up a class. It might not be necessarily clear to students `solution_script` is an object instance so they might not treat the file like a class.

Reply:
Sounds good :) I think this exercise just needs a few indentation fixes, and we're ready to go ;)

Reply:
Actually, it might be confusing to have the nested indenting because a student might not realize they need to maintain the indentation as well. We should just stick to using the instance of a script without a nested class.